x-pack/plugins/observability_solution/apm/public/components/app/service_tab_empty_state/index.tsx

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/* eslint-disable @elastic/eui/href-or-on-click */
```

Comment:
>[!NOTE]
The `EuiButton` currently has a type restriction that allows either the `href` or `onClick` props, but not both simultaneously. 
Last month, @cauemarcondes  briefly discussed this with the EUI team, and they understand the need to use both props simultaneously. They will be making the necessary modifications in the future. In the meantime, the workaround is to disable the rule, similar to the approach taken in [this other PR](https://github.com/elastic/kibana/pull/189704).